hummingbird/ml/operator_converters/xgb.py

```diff
@@ -103,6 +103,23 @@ def convert_sklearn_xgb_classifier(operator, device, extra_config):
     return convert_gbdt_classifier_common(tree_infos, _get_tree_parameters, n_features, n_classes, extra_config=extra_config)
 
 
+def convert_sklearn_xgb_ranker(operator, device, extra_config):
```

Comment:
I think that we can just call `convert_sklearn_xgb_regressor` from `register_converter`.

Reply:
I thought it'd be easier to read this way. Anyway, let me change that.